tsconfig.json

```diff
@@ -7,7 +7,7 @@
     "experimentalDecorators": true,
     "jsx": "react",
     "lib": ["dom", "es2015", "es2016", "es2017"],
-    "module": "es2015",
+    "module": "esnext",
```

Comment:
This is related to the `import` promise function, yah? One thing to keep in mind is that we compile out Reaction using Babel 7 so there needs to be some testing around that should we decide to keep.